Utils/azuretools-core/src/com/microsoft/azuretools/azurecommons/rediscacheprocessors/RedisCacheCreator.java

```diff
@@ -59,8 +59,8 @@ private void initCreatorsForPremiumTier(RedisCaches redisCaches, String dnsName,
     	}
     }
     public RedisCacheCreator(RedisCaches redisCaches, String dnsName, String regionName, String groupName) {
-    	initCreatorsForBasicTier(redisCaches, dnsName, regionName, groupName, new int[] {0, 1, 2, 3, 4, 6});
-    	initCreatorsForStdTier(redisCaches, dnsName, regionName, groupName, new int[] {0, 1, 2, 3, 4, 6});
+    	initCreatorsForBasicTier(redisCaches, dnsName, regionName, groupName, new int[] {0, 1, 2, 3, 4, 5, 6});
+    	initCreatorsForStdTier(redisCaches, dnsName, regionName, groupName, new int[] {0, 1, 2, 3, 4, 5, 6});
```

Comment:
Actually as we have discussed before, I'm going to change the redis cache creation logic in next sprint. So I guess this code snippet will disappear in future